darts/timeseries.py

```diff
@@ -705,6 +919,28 @@ def from_pickle(cls, path: str) -> "TimeSeries":
     ==========
     """
 
+    @property
+    def static_covariates(self) -> Optional[pd.DataFrame]:
+        """
+        Returns the static covariates contained in the series as a pandas DataFrame.
+        The columns represent the static variables and the rows represent the components of the uni/multivariate
+        series. If a single-row DataFrame, the covariates are globally 'applied' to all components of the
```

Comment:
Do we need to keep this sentence and the rest of the docstring below? seems copy-pasted from the constructor.